File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.kt
class Solution {
    fun solution(num_list: IntArray): Int {
        var answer: Int = 0
        
        var mul = 1
        var sum_s = num_list.sumOf{it} * num_list.sumOf{it}
        
        for (i in num_list){
            mul *= i
        }
        
        if (mul < sum_s){
            answer = 1
        }
        return answer
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
# 원소들의 곱과 합
# 모든 원소들의 곱이 모든 원소들의 합의 제곱보다 작으면 1
# 아닌 경우 0을 return

def solution(num_list):
    answer = 0
    
    mul = 1
    sum_s = sum(num_list) * sum(num_list)
    
    for i in num_list:
        mul *= i

    if (mul < sum_s):
        answer = 1
    
    
    return answer